doc/source/whatsnew/v0.25.0.rst

```diff
@@ -236,6 +236,7 @@ Groupby/Resample/Rolling
 - Bug in :meth:`pandas.core.groupby.DataFrameGroupBy.nunique` in which the names of column levels were lost (:issue:`23222`)
 - Bug in :func:`pandas.core.groupby.GroupBy.agg` when applying a aggregation function to timezone aware data (:issue:`23683`)
 - Bug in :func:`pandas.core.groupby.GroupBy.first` and :func:`pandas.core.groupby.GroupBy.last` where timezone information would be dropped (:issue:`21603`)
+- Replaced regular dictionaries with ``OrderedDict`` in the aggregation functions of ``groupby`` to make the ordering consistent in all versions of Python (:issue:`25692`)
```

Comment:
Write this as a user-facing message. I suspect they don't care about the internals of groupby. So something like

"Ensured consistent ordering of outputs in groupby aggregation functions on all versions of Python"

Reply:
Do you think that maybe it would be better to do a two part statement? To give a user-facing message, as well as some technical aspect. For example:

- Ensured that ordering of outputs in ``groupby`` aggregation functions is consistent across all versions of Python. This was achieved through replacement of regular dictionaries with ``OrderedDict`` in the aggregation functions of ``groupby`` (:issue:`25692`)